src/components/CMSForm/index.tsx

```diff
@@ -48,14 +51,25 @@ const RenderForm = ({ form }: { form: Form }) => {
         }, 1000)
 
         try {
+          const hubspotCookie = getCookie('hubspotutk')
+          const pageUri = `${process.env.NEXT_PUBLIC_SITE_URL}${pathname}`
+          const slugParts = pathname?.split('/')
+          const pageName =
+            slugParts && slugParts[slugParts.length - 1] === ''
```

Comment:
It feels like this logic could be cleaned up a bit. Something like `slugParts?.at(-1) === ''`